packages/server/test/open-feature.spec.ts

```diff
@@ -85,11 +85,11 @@ describe('OpenFeature', () => {
       OpenFeature.setProvider(provider);
       OpenFeature.setProvider(domain, fakeProvider);
 
-      const defaultClient = OpenFeature.getClient();
-      const domainSpecificClient = OpenFeature.getClient(domain);
+      const defaultProvider = OpenFeature.getProvider();
+      const domainSpecificProvider = OpenFeature.getProvider(domain);
 
-      expect(defaultClient.metadata.providerMetadata.name).toEqual(provider.metadata.name);
-      expect(domainSpecificClient.metadata.providerMetadata.name).toEqual(fakeProvider.metadata.name);
+      expect(defaultProvider).toEqual(provider);
+      expect(domainSpecificProvider).toEqual(fakeProvider);
     });
 
     it('should bind a new provider to existing clients in a matching domain', () => {
```

Comment:
This test could also be improved if we add a method on the client to get the provider. I'm just not sure if it's a good idea.